src/dataframe/dataframe.jl

```diff
@@ -279,7 +284,8 @@ end
 function Base.getindex(df::DataFrame, row_ind::Real, col_inds::AbstractVector)
     selected_columns = index(df)[col_inds]
     new_columns = Any[dv[[row_ind]] for dv in columns(df)[selected_columns]]
-    return DataFrame(new_columns, Index(_names(df)[selected_columns]))
+    # no subsetting required for metadata cause rows dont matter
```

Comment:
This comment doesn't sound very useful, meta-data just works as the index.

Reply:
fixed